keyboards/keycapsss/plaid_pad/keymaps/via/config.h

```diff
@@ -0,0 +1,3 @@
+#define VIAL_KEYBOARD_UID {0xA2, 0xF1, 0x8A, 0x6F, 0xB1, 0xA5, 0x01, 0x03}
```

Comment:
Remove this file.

Reply:
Can we keep this? It has no negative effect.
Then it easier to build firmware on the vial fork.

Reply:
If VIAL is upstreamed to QMK master we may consider it, but at this stage, no.

Reply:
Done removed it.